src/test/java/hu/elte/eserial/util/TypeUtilsTest.java

```diff
@@ -165,4 +166,181 @@ public void isCharacter_GivenCharacter_ReturnsTrue() {
     public void isCharacter_GivenNonCharacter_ReturnsFalse() {
         assertFalse(TypeUtils.isCharacter(Integer.class));
     }
+
+    @Test
+    public void isDecimal_GivenDecimal_ReturnsTrue() {
+        assertTrue(TypeUtils.isDecimal(double.class));
+        assertTrue(TypeUtils.isDecimal(Double.class));
+        assertTrue(TypeUtils.isDecimal(float.class));
+        assertTrue(TypeUtils.isDecimal(Float.class));
+    }
+
+    @Test
+    public void isDecimal_GivenNonDecimal_ReturnsFalse() {
+        assertFalse(TypeUtils.isDecimal(Integer.class));
+    }
+
+    @Test
+    public void isLong_GivenLong_ReturnsTrue() {
+        assertTrue(TypeUtils.isLong(Long.class));
+        assertTrue(TypeUtils.isLong(long.class));
+    }
+
+    @Test
+    public void isLong_GivenNonLong_ReturnsFalse() {
+        assertFalse(TypeUtils.isLong(Integer.class));
+    }
+
+    @Test
+    public void isAssignableFrom_GivenCollection_ReturnsTrue() {
```

Comment:
The method names don't make much sense here, the other parameter's value is missing.